src/routes/index.ts

```diff
@@ -77,6 +77,20 @@ export class Routes {
 				.catch((error) => InternalServerError.send(res, error));
 		});
 
+		app.get('/nodes/nodePublicKey/:nodePublicKey', (req: Request, res: Response) => {
```

Comment:
do you think we should rename `nodePublicKey ` -> `transportPublicKey`?

I think we need change on Wallet and Explorer as well?

Reply:
Catapult-REST returns it as `nodePublicKey`. IMO, it should be consistent.

Reply:
But if we decide to change the name, it should be also applied to Catapult-REST.

Reply:
>Catapult-REST returns it as nodePublicKey. IMO, it should be consistent.

Agree, that's why I kept the name.